### PR TITLE
Update credential documentation

### PR DESCRIPTION
**Pull request by mjendrysik-hpe**
## Summary and Scope

Update to password change documentation to avoid out-of-order operation resulting in the inability to power components on and off.

## Issues and Related PRs

* Resolves [CASMHMS-5472](https://jira-pro.its.hpecorp.net:8443/browse/CASMHMS-5472) for mainline
* Change will also be needed in `release/1.2`, `release/1.1`, `release/1.0`

## Pull Request Checklist

- [x] Target branch correct

### CODE REVIEW

**Review by rsjostrand-hpe — APPROVED** [review]

File: operations/security_and_authentication/Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md [hunk 1]
```diff
@@ -11,13 +11,37 @@ This procedure changes the global default credential on HPE Cray EX liquid-coole
...
-   ```bash
-   ncn-m001# sat bootsys shutdown --stage bos-operations \
-   --bos-templates COS_SESSION_TEMPLATE
+   ```screen
+   ncn-m001# sat bootsys shutdown --stage bos-operations --bos-templates COS_SESSION_TEMPLATE
    ```
-2. Perform procedures in [Provisioning a Liquid-Cooled EX Cabinet CEC with Default Credentials](Provisioning_a_Liquid-Cooled_EX_Cabinet_CEC_with_Default_Credentials.md).
 
-3. Perform procedures in [Updating the Liquid-Cooled EX Cabinet Default Credentials after a CEC Password Change](Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md).
+2. Disable the hms-discovery Kubernetes cron job.
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md:18]:
```suggestion
2. Disable the `hms-discovery` Kubernetes cron job.
```

File: operations/security_and_authentication/Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md [hunk 2]
```diff
@@ -11,13 +11,37 @@ This procedure changes the global default credential on HPE Cray EX liquid-coole
...
+   ncn-m001# cray capmc xname_off create --xnames x[1000-1003]c[0-7]s[0-7] --format json
+   ```
+
+   Check the power status:
+
+   ```screen
+   ncn-m001# cray capmc get_xname_status create --xnames x[1000-1003]c[0-7]s[0-7] --format json
+   ```
+
+   Continue when all compute slots are Off.
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md:40]:
```suggestion
   Continue when all compute slots are `Off`.
```

File: operations/security_and_authentication/Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md [hunk 3]
```diff
@@ -11,13 +11,37 @@ This procedure changes the global default credential on HPE Cray EX liquid-coole
...
+
+4. Perform procedures in [Provisioning a Liquid-Cooled EX Cabinet CEC with Default Credentials](Provisioning_a_Liquid-Cooled_EX_Cabinet_CEC_with_Default_Credentials.md).
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md:42]:
```suggestion
4. Perform the procedures in [Provisioning a Liquid-Cooled EX Cabinet CEC with Default Credentials](Provisioning_a_Liquid-Cooled_EX_Cabinet_CEC_with_Default_Credentials.md).
```

File: operations/security_and_authentication/Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md [hunk 4]
```diff
@@ -11,13 +11,37 @@ This procedure changes the global default credential on HPE Cray EX liquid-coole
...
+
+5. Perform procedures in [Updating the Liquid-Cooled EX Cabinet Default Credentials after a CEC Password Change](Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md).
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md:44]:
```suggestion
5. Perform the procedures in [Updating the Liquid-Cooled EX Cabinet Default Credentials after a CEC Password Change](Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md).
```

File: operations/security_and_authentication/Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Change Cray EX Liquid-Cooled Cabinet Global Default Password
 
-This procedure changes the global default credential on HPE Cray EX liquid-cooled cabinet embedded controllers (BMCs). The chassis management module (CMM) controller (cC), node controller, and Slingshot switch controller (sC) are generically referred to as "BMCs" in these procedures.
+This procedure changes the global default `root` credential on HPE Cray EX liquid-cooled cabinet embedded controllers (BMCs). The chassis management module (CMM) controller (cC), node controller (nC), and Slingshot switch controller (sC) are generically referred to as "BMCs" in these procedures.
 
 ### Prerequisites
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md:5]:
```suggestion
## Prerequisites
```

File: operations/security_and_authentication/Provisioning_a_Liquid-Cooled_EX_Cabinet_CEC_with_Default_Credentials.md [hunk 6]
```diff
@@ -82,7 +84,7 @@ This procedure does not provision Slingshot switch BMCs. Slingshot switch BMC de
 
 9. Enter `set_hash` and provide the password hash value as the argument.
 
-   The CEC validates the input syntax of the hash. Adding an extra char or omitting a character is flagged as an error. I a character is changed, the password entered in the serial console login shell or the Redfish `root` account will not work. If that happens, rerun the `set_hash` command on the CEC and reboot the CMMs.
+   The CEC validates the input syntax of the hash. Adding an extra char or omitting a character is flagged as an error. If a character is changed, the password entered in the serial console login shell or the Redfish `root` account will not work. If that happens, rerun this procedure from the beginning.
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Provisioning_a_Liquid-Cooled_EX_Cabinet_CEC_with_Default_Credentials.md:87]:
```suggestion
   The CEC validates the input syntax of the hash. Adding an extra character or omitting a character is flagged as an error. If a character is changed, the password entered in the serial console login shell or the Redfish `root` account will not work. If that happens, then rerun this procedure from the beginning.
```

File: operations/security_and_authentication/Provisioning_a_Liquid-Cooled_EX_Cabinet_CEC_with_Default_Credentials.md [hunk 7]
```diff
@@ -107,31 +109,13 @@ This procedure does not provision Slingshot switch BMCs. Slingshot switch BMC de
...
-       ```
-
-       Power on the compute chassis slots:
-
-       ```bash
-       ncn-m001# cray capmc xname_on create --xnames x[1000-1003]c[0-7]s[0-7] --format json
-       ```
-
-    2. If the cabinet has not been provisioned with CSM or other management software (bare-metal), the compute chassis slots are most likely powered off. To perform chassis power control operations, SSH to a CMM and and use the `redfish -h` command to display the power control commands:
+    1. To perform blade power control operations, SSH to a CMM and and use the `redfish` command to perform the power cycle. This must be done for each populated compute blade in each odd or even numbered chassis in the cabinet depending on which CEC issued the reset above.
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Provisioning_a_Liquid-Cooled_EX_Cabinet_CEC_with_Default_Credentials.md:116]:
```suggestion
    1. To perform blade power control operations, SSH to a CMM and and use the `redfish` command to perform the power cycle. This must be done for each populated compute blade in each odd- or even-numbered chassis in the cabinet, depending on which CEC issued the reset above.
```

File: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md [hunk 8]
```diff
@@ -4,10 +4,12 @@ This procedure changes the credential for liquid-cooled EX cabinet chassis contr
 
 **NOTE:** This procedure does not provision Slingshot switch BMCs (RouterBMCs). Slingshot switch BMC default credentials must be changed using the procedures in the Slingshot product documentation. To update Slingshot switch BMCs, refer to "Change Rosetta Login and Redfish API Credentials" in the *Slingshot Operations Guide (> 1.6.0)*.
 
-This procedure provisions only the default Redfish root account passwords. It does not modify Redfish accounts that have been added after an initial system installation.
+This procedure provisions only the default Redfish `root` account passwords. It does not modify Redfish accounts that have been added after an initial system installation.
 
 ### Prerequisites
 
+- The hms-discovery Kubernetes cron job has been disabled.
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md:11]:
```suggestion
- The `hms-discovery` Kubernetes cron job has been disabled.
```

File: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md [hunk 9]
```diff
@@ -210,12 +211,18 @@ Before redeploying MEDS, update the `customizations.yaml` file in the `site-init
     > ncn-m001# cray hsm inventory redfishEndpoints update BMC_XNAME --user root --password ${CRED_PASSWORD}
     > ```
 
-3. Wait for HSM to re-discover the updated RedfishEndpoints:
+3. Restart the hms-discovery Kubernetes cron job.
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md:214]:
```suggestion
3. Restart the `hms-discovery` Kubernetes cron job.
```

File: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md [hunk 10]
```diff
@@ -210,12 +211,18 @@ Before redeploying MEDS, update the `customizations.yaml` file in the `site-init
...
+   ```bash
+   ncn-m001# kubectl -n services patch cronjobs hms-discovery -p '{"spec" : {"suspend" : false }}'
+   ```
+   After 2-3 minutes the hms-discovery cron job will start to power on all of the currently powered off compute slots.
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md:218]:
```suggestion
   After 2-3 minutes, the `hms-discovery` cron job will start to power on all of the currently powered off compute slots.
```

File: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md [hunk 11]
```diff
@@ -210,12 +211,18 @@ Before redeploying MEDS, update the `customizations.yaml` file in the `site-init
...
+
+4. Wait for compute slots to be pwoered on and for HSM to re-discover the updated RedfishEndpoints:
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md:220]:
```suggestion
4. Wait for compute slots to be pwoered on and for HSM to re-discover the updated Redfish endpoints.
```

File: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md [hunk 12]
```diff
@@ -210,12 +211,18 @@ Before redeploying MEDS, update the `customizations.yaml` file in the `site-init
...
     ```bash
-    ncn-m001# sleep 180
+    ncn-m001# sleep 300
     ```
 
-4. Wait for all updated Redfish endpoints to become `DiscoverOK`:
+5. Wait for all updated Redfish endpoints to become `DiscoverOK`:
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md:225]:
```suggestion
5. Wait for all updated Redfish endpoints to become `DiscoverOK`.
```

File: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md [hunk 13]
```diff
@@ -210,12 +211,18 @@ Before redeploying MEDS, update the `customizations.yaml` file in the `site-init
...
 
     The following bash script will find all Redfish endpoints for the liquid-cooled BMCs that are not in `DiscoverOK`, and display their last Discovery Status.
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md:227]:
```suggestion
    The following bash script will find all Redfish endpoints for the liquid-cooled BMCs that are not in `DiscoverOK`, and display their `LastDiscoveryStatus`.
```

**Review by mharding-hpe — DISMISSED** [review]
After the PR receives the necessary approvals, backport PRs will need to be created for the release branches.

**Review by nrockershousen — APPROVED** [review]

File: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md [hunk 14]
```diff
@@ -210,14 +211,20 @@ Before redeploying MEDS, update the `customizations.yaml` file in the `site-init
...
     ```bash
-    ncn-m001# sleep 180
+    ncn-m001# sleep 300
     ```
 
-4. Wait for all updated Redfish endpoints to become `DiscoverOK`:
+5. Wait for all updated Redfish endpoints to become `DiscoverOK`.
 
-    The following bash script will find all Redfish endpoints for the liquid-cooled BMCs that are not in `DiscoverOK`, and display their last Discovery Status.
+    The following bash script will find all Redfish endpoints for the liquid-cooled BMCs that are not in `DiscoverOK`, and display their `lastDiscoveryStatus`.
```
Comment (mharding-hpe) [inline operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md:227]:
```suggestion
    The following Bash script will find all Redfish endpoints for the liquid-cooled BMCs that are not in `DiscoverOK`, and display their `lastDiscoveryStatus`.
```

**Review by mharding-hpe — APPROVED** [review]

File: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md (hunk 14 above)
Comment (mjendrysik-hpe) [reply operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md:227]: Grr, I changed that!

**Pull request merged**